.github/workflows/ci-pipeline.yml

```
@@ -4,8 +4,10 @@ concurrency: ci-shotcaller
 permissions:
   contents: read
   packages: write
+  gists: write
```

actionlint (1.7.7)

unknown permission scope "gists". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)